databricks_cli/utils.py

```diff
@@ -161,3 +161,14 @@ def for_profile(profile):
              'Please configure by entering '
              '`{argv} configure --profile {profile}`').format(
                 profile=profile, argv=sys.argv[0]))
+
+
+def merge_dicts(*dicts):
+    """
+    Merges dicts throw shallow copy.
```

Comment:
*through

Please rename this to `merge_dicts_shallow` to reflect the shallow nature at the call site.

Reply:
No need for this function at all, per the other comment.

Reply:
Per other comment